index.html

```diff
@@ -75,7 +75,7 @@ <h1 class="head__title fz-40 fw-700 lh-52">
                     </li>
                     <li class="head__icons__li">
                         <a href="#" class="head__icons__link link"><img
-                                src="images/hand.svg" alt="hand icon" /></a>
+                                src="images/hand.svg" alt="Angilist icon" /></a>
```

Comment:
- [ ] Please remove the text "icon", even don't use the words like `button`, `icon`, `image of` in your **alternative-text** because by default when  a person who use screen-readers, screen reader automatically provide the context. Kindly use "Angelist Profile" because it will assist the person to know whether he/she is interested to look into your `profile` or not.